lib/iris/coords.py

```diff
@@ -1043,9 +1066,9 @@ def _sanity_check_contiguous(self):
                                  'coordinates with 4 bounds.'.format
                                  (self.name(), self.nbounds))
         else:
-            raise ValueError('Invalid operation for {!r}. Contiguous bounds '
-                             'are not defined for coordinates with more than '
-                             '2 dimensions.'.format(self.name()))
+            raise ValueError('Invalid operation for {!r}. Not supported for '
+                             'bounds with more than 2 dimensions.'.format
+                             (self.name()))
```

Comment:
I'm not sure about this error message.  It seems slightly more confusing than it was previously.

Reply:
I think I changed this because I wanted the method `_sanity_check_bounds` to be able to be used for other things, but the previous error message.
Saying that, I realise I have made a lot of breaking changes in this PR, the above being one of them so I am going to undo that and put the error message back